src/main/java/com/matt/forgehax/mods/ActiveModList.java

```diff
@@ -69,6 +72,15 @@
           .defaultTo(SortMode.ALPHABETICAL)
           .build();
   
+  private final Setting<Boolean> colourSetting =
+      getCommandStub()
+          .builders()
+          .<Boolean>newSettingBuilder()
+          .name("colourMode")
```

Comment:
`colour` to `color` please (sorry redcoats).

Also change `.name("colourMode")` to `.name("color-mode")`